mlir/python/pyproject.toml

```diff
@@ -0,0 +1,40 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Copyright (c) 2025.
+
+[project]
+name = "mlir-python-bindings"
+version = "0.0.1"
+requires-python = ">=3.8,<=3.13"
```

Comment:
Is there anything that prevents the mlir bindings from working with Python versions newer than 3.13?

Reply:
no - this toml is copy-pasta from some downstream uses where I do have a hard upperbound but for upstream there's no blocker.

Reply:
Should we remove it then?